### PR TITLE
instruction update for java devs

### PR DESCRIPTION
**Pull request by Sbonelo01**
## Description:
Some java developers only submit the function in an attempt to follow the instructions. This PR tries to address that.

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/katas/level-1/_index.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ If you are writing JS code then your files would follow a similar pattern. You w
 ```js
 module.exports = { YOUR_FUNCTION_NAME };
 ```
+## Note to Java developers:
+
+- Even though the instructions will specifically ask for a function, please don't submit only the function/method. Make sure that there is a wrapping public class in your code.
```
Comment (sheenarbw) [inline content/projects/katas/level-1/_index.md]: Please add examples. 

The learners will either put everything in the same class or spread all the functions out across multiple files.  The learners will end up guessing.  It's important to be very very explicit about what is required here. 

Please reach out to @ruddy-m  to ask her if you aren't sure how the code should be structured.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
It's a good try, but not specific enough.

**Comment by Sbonelo01** [comment]
@ruddy-m , please review when you can

File: content/projects/katas/level-1/_index.md [hunk 2]
```diff
@@ -61,6 +61,31 @@ If you are writing JS code then your files would follow a similar pattern. You w
 ```js
 module.exports = { YOUR_FUNCTION_NAME };
 ```
+## Note to Java developers:
+
+- Even though the instructions will specifically ask for a function, please don't submit only the function/method. Make sure that there is a wrapping public class for your function/method.
```
Comment (sheenarbw) [inline content/projects/katas/level-1/_index.md]: Why does this need a bullet point?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Some strange text formatting, otherwise it's fine

File: content/projects/katas/level-1/_index.md [hunk 3]
```diff
@@ -236,4 +261,4 @@ Also, the output lists should maintain the list items in the original order in w
 
 ### Exports
 
-- **For those using Javascript:** Make sure they export their functions correctly. Otherwise, the marking bot won't be able to access their functions.
\ No newline at end of file
+- **For those using Javascript:** Make sure they export their functions correctly. Otherwise, the marking bot won't be able to access their functions.
```
Comment (ruddy-m) [inline content/projects/katas/level-1/_index.md:264]: this should also be JavaScript

**Review by ruddy-m — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**